diff --git a/client/pages/Home/components.js b/client/pages/Home/components.js
index 8984c0f442e08fdcc9e4413fcc386e60dcef7145..7a9e954ea62b52c8872477cd8e079e1623d582b6 100644
--- a/client/pages/Home/components.js
+++ b/client/pages/Home/components.js
@@ -4,6 +4,7 @@ import { Fragment } from 'react';
 import { Link } from 'react-router-dom';
 import { withPseudoState } from 'react-pseudo-state';
 import { useList } from '../../providers/List';
+import { useAdminMeta } from '../../providers/AdminMeta';
 
 import CreateItemModal from '../../components/CreateItemModal';
 
@@ -37,6 +38,7 @@ const BoxElement = props => (
 );
 
 const BoxComponent = ({ focusOrigin, isActive, isHover, isFocus, meta, ...props }) => {
+  const { showDashboardCounts } = useAdminMeta();
   const { list, openCreateItemModal } = useList();
   const { label, singular } = list;
 
@@ -52,7 +54,7 @@ const BoxComponent = ({ focusOrigin, isActive, isHover, isFocus, meta, ...props
         >
           {label}
         </Name>
-        <Count meta={meta} />
+        {showDashboardCounts && <Count meta={meta} />}
         <CreateButton
           title={`Create ${singular}`}
           isHover={isHover || isFocus}
diff --git a/client/pages/Home/index.js b/client/pages/Home/index.js
index 6e4f9ac3e08eb3aab3f5feba6187b360e7f2984d..12919b91ddab3634d6e015adf230d53c9ba67568 100644
--- a/client/pages/Home/index.js
+++ b/client/pages/Home/index.js
@@ -25,7 +25,7 @@ const getCountQuery = lists => {
 };
 
 const Homepage = () => {
-  const { getListByKey, listKeys, adminPath } = useAdminMeta();
+  const { getListByKey, listKeys, adminPath, showDashboardCounts } = useAdminMeta();
 
   // TODO: A permission query to limit which lists are visible
   const lists = listKeys.map(key => getListByKey(key));
@@ -33,6 +33,7 @@ const Homepage = () => {
   const { data, error } = useQuery(getCountQuery(lists), {
     fetchPolicy: 'cache-and-network',
     errorPolicy: 'all',
+    skip: !showDashboardCounts,
   });
 
   const [cellWidth, setCellWidth] = useState(3);
diff --git a/index.js b/index.js
index 8a2efda8eb3c33d2791466db161c503987fc4f91..43f9f4ece2394c71ba97577125efe2f73aaf78fd 100644
--- a/index.js
+++ b/index.js
@@ -24,6 +24,7 @@ class AdminUIApp {
     adminMeta = {},
     defaultPageSize = 50,
     maximumPageSize = 1000,
+    showDashboardCounts = true,
   } = {}) {
     if (adminPath === '/') {
       throw new Error("Admin path cannot be the root path. Try; '/admin'");
@@ -56,6 +57,7 @@ class AdminUIApp {
       signinPath: `${this.adminPath}/signin`,
       signoutPath: `${this.adminPath}/signout`,
     };
+    this.showDashboardCounts = showDashboardCounts;
   }
 
   isAccessAllowed(req) {
@@ -111,7 +113,7 @@ class AdminUIApp {
 
   getAdminUIMeta(keystone) {
     // This is exposed as the global `KEYSTONE_ADMIN_META` in the client.
-    const { name, adminPath, apiPath, graphiqlPath, pages, hooks } = this;
+    const { name, adminPath, apiPath, graphiqlPath, pages, hooks, showDashboardCounts } = this;
     const { signinPath, signoutPath } = this.routes;
     const { lists } = keystone.getAdminMeta({ schemaName: this._schemaName });
     const authStrategy = this.authStrategy ? this.authStrategy.getAdminMeta() : undefined;
@@ -149,6 +151,7 @@ class AdminUIApp {
       authStrategy,
       lists,
       name,
+      showDashboardCounts,
       ...this._adminMeta,
     };
   }
