diff --git a/index.js b/index.js
index a24a43d2503f34eff529941303be82efeef7d137..bd1705055c10ba10669e3a401125d2674acecb36 100644
--- a/index.js
+++ b/index.js
@@ -1,5 +1,5 @@
 const express = require('express');
-const { graphqlUploadExpress } = require('graphql-upload');
+const graphqlUploadExpress = require('graphql-upload/graphqlUploadExpress.js');
 const { GraphQLPlaygroundApp } = require('@keystonejs/app-graphql-playground');
 const validation = require('./validation');
 
