diff --git a/dist/access-control.cjs.dev.js b/dist/access-control.cjs.dev.js
index 796e38e680622a0669119c230f7b056a0216f573..64f20c9f029128626b2d1b29b8f480b98a0301f4 100644
--- a/dist/access-control.cjs.dev.js
+++ b/dist/access-control.cjs.dev.js
@@ -266,7 +266,8 @@ async function validateListAccessControl({
   gqlName,
   itemId,
   itemIds,
-  context
+  context,
+  args,
 }) {
   // Either a boolean or an object describing a where clause
   let result = false;
@@ -283,7 +284,8 @@ async function validateListAccessControl({
       gqlName,
       itemId,
       itemIds,
-      context
+      context,
+      args
     });
   }
 
diff --git a/dist/access-control.cjs.prod.js b/dist/access-control.cjs.prod.js
index 796e38e680622a0669119c230f7b056a0216f573..a3a78c6bc0231e670d1e731800b3828ee9640bc0 100644
--- a/dist/access-control.cjs.prod.js
+++ b/dist/access-control.cjs.prod.js
@@ -266,7 +266,8 @@ async function validateListAccessControl({
   gqlName,
   itemId,
   itemIds,
-  context
+  context,
+  args,
 }) {
   // Either a boolean or an object describing a where clause
   let result = false;
@@ -283,7 +284,8 @@ async function validateListAccessControl({
       gqlName,
       itemId,
       itemIds,
-      context
+      context,
+      args,
     });
   }
 
diff --git a/dist/access-control.esm.js b/dist/access-control.esm.js
index 1ef09a1d54a7eb0a1e2d20aa41fba143c0293c17..3ebac9df658fda768d2e05fd61e1fe3004e8e602 100644
--- a/dist/access-control.esm.js
+++ b/dist/access-control.esm.js
@@ -262,7 +262,8 @@ async function validateListAccessControl({
   gqlName,
   itemId,
   itemIds,
-  context
+  context,
+  args,
 }) {
   // Either a boolean or an object describing a where clause
   let result = false;
@@ -279,7 +280,8 @@ async function validateListAccessControl({
       gqlName,
       itemId,
       itemIds,
-      context
+      context,
+      args,
     });
   }
 
